### ISSUE TITLE
/entrypoint.sh: 26: exec:  bitcoin-cli: not found error

### ISSUE THREAD

**Issue by abhyankar-amey** [ruimarinho/docker-bitcoin-core#86]
Hello,

When I execute the docker run to get balance, I am getting following error = 

`docker run -it --rm btc-18nov \ bitcoin-cli \ -rpcconnect=bitcoin-server \-rpcuser=amey\ -regtest=1 \ -stdinrpcpass \ getbalance
/entrypoint.sh: 4: [: =: unexpected operator
/entrypoint.sh: 10: [: =: unexpected operator

/entrypoint.sh: 26: exec:  bitcoin-cli: not found
`
Pls suggest how to fix this issue thanks.

**Comment by ruimarinho** [comment]
@abhyankar-amey it looks like you're not running the image produced by this project?

**Comment by abhyankar-amey** [comment]
@ruimarinho 

I am trying from last couple of weeks with the default image,customisation and lot other stuff.
Errors & errors.

I am using = 

Docker version 19.03.4, build 9013bf583a
Base machine: Ubuntu 18.04.3 LTS

If I start the container using default values & then if I try to send RPC call as per the readme.md, it says make sure server is listening on respective port.
I generate the password using the .py file you mentioned in the readme.

Same error if I am using regtest or prod network.
I ensured that bitcoind is running inside the container.
I tried to telnet on the port but it doesnt work.
For other containers, I am able to telnet without any issues.

I also notice, that 'rpcallowip'  option is no more allowed in 0.18.x onwards = https://bitcoin.org/en/release/v0.18.0#notable-changes

If I try to get output by doing docker exec, it works.
It's the RPC call which doesn't work even if I supply custom bitcoin.conf file.
